server/py/framework/db/sqldb/db.py

```diff
@@ -60,11 +60,16 @@
 import mlrun.model
 import mlrun.utils.db
 from mlrun.artifacts.base import fill_artifact_object_hash
+from mlrun.common.schemas import FunctionState
 from mlrun.common.schemas.feature_store import (
     FeatureSetDigestOutputV2,
     FeatureSetDigestSpecV2,
 )
-from mlrun.common.schemas.model_monitoring import EndpointType, ModelEndpointSchema
+from mlrun.common.schemas.model_monitoring import (
+    EndpointType,
+    ModelEndpointSchema,
+    constants,
```

Comment:
`constants` can be anything, and there is no need to import the whole module while you only need `ModelMonitoringAppLabel` and `ModelMonitoringInfraLabel`
```suggestion
```